src/sm.c

```diff
@@ -120,7 +121,7 @@ int sm_open_begin(SM_HANDLE sm)
     else
     {
         /*Codes_SRS_SM_02_039: [ If the state is not SM_CREATED then sm_open_begin shall return SM_EXEC_REFUSED. ]*/
-        LONG state = InterlockedAdd(&sm->state, 0);
+        LONG state = interlocked_add(&sm->state, 0);
```

Comment:
>LONG [](start = 8, length = 4)

int32_t

Reply:
Fixed.

---
In reply to: [501160920](https://github.com/Azure/azure-c-util/pull/66#discussion_r501160920) [](ancestors = 501160920)